tests/common/test_datasets_utils.py

```diff
@@ -21,6 +21,11 @@ def test_translate(input_str, vocab, output_str):
     assert out == output_str
 
 
+def test_translate_unknown_vocab():
+    with pytest.raises(KeyError):
+        _ = utils.translate("test", "unknown_vocab")
```

Comment:
```suggestion
    with pytest.raises(KeyError):
        utils.translate("test", "unknown_vocab")
```
same suggestion for the other contexts